bulk-rnaseq.yaml

```diff
@@ -123,3 +123,11 @@ tools:
   owner: iuc
   tool_panel_section_label: RNA-Seq
   tool_shed_url: toolshed.g2.bx.psu.edu
+- name: fastq_quality_trimmer
```

Comment:
These are generic fastq handling tools rather than necessarily RNA-seq specific. Could you maybe a identify a more generic categorisation in the UI? The same might go for the tools above.

Reply:
Done in 02cf0096eb7d8087f6afdeef12bf7454f7f23822, but requires that [changes to atlas-lsf-ebi-ansible repo](https://gitlab.ebi.ac.uk/ebi-gene-expression/galaxy-lsf-ebi-ansible/-/merge_requests/8) be merged for new tool panel section names to become valid.